packages/management/addons-manager/bundle/vendir.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+minimumRequiredVersion: 0.12.0
+directories:
+  - path: bundle/config
+    contents:
+      - path: upstream
+        manual: {}
+      - path: overlays
+        manual: {}
+      - path: values.yaml
+        manual: {}
+  - path: package.yaml
```

Comment:
this is not needed, as package.yaml and metadata.yaml are generated when you scaffold a package

Reply:
removed package.yaml and metadata.yaml in vendir.yml